jenkins/build.jpl

```diff
@@ -98,7 +99,9 @@ node("builder") {
     CPU arch:  ${params.ARCH}
     Describe:  ${params.GIT_DESCRIBE}
     Revision:  ${params.COMMIT_ID}
-    Config:    ${params.DEFCONFIG}""")
+    Config:    ${params.DEFCONFIG}
+    Compiler:  ${params.COMPILER}
```

Comment:
Might be better to just put the full Docker image name on one line to be clear which once was used.  Also having its ID number would help with reproducing builds with the exact same one.

Reply:
Good idea, will do